webapp-django/crashstats/crashstats/models.py

```diff
@@ -1130,6 +1131,16 @@ class ProductBuildTypes(SocorroMiddleware):
     )
 
 
+class ProductIDMap(SocorroMiddleware):
+    cache_seconds = 1
```

Comment:
As far as I can tell, we only change that table via migrations which require a deploy. I think we could increase the cache to something higher.